src/apiServices/AuthServiceNode.ts

```diff
@@ -42,11 +41,12 @@ export class AuthServiceNode {
     public async logout() {
         if (this.isUserLoggedIn()) {
             try {
-                const { access } = getTokensNode(tokenLocation)
-                const headers = getHeaders(access)
+                const headers = getHeaders(this.jwtTokens.access)
 
-                await postRequest(`${API_URL_PRODUCTION}/auth/logout`, {}, headers)
-                removeTokensNode(tokenLocation)
+                const result = await postRequest(`${API_URL_PRODUCTION}/auth/logout`, {}, headers)
+                this.jwtTokens.access, this.jwtTokens.refresh = ''
```

Comment:
I think this needs to be `this.jwtTokens.access = this.jwtTokens.refresh = ''`

Reply:
oh yes nice catch!!